confgenerator/logging_receivers.go

```diff
@@ -682,6 +682,63 @@ func (r LoggingReceiverSystemd) Components(ctx context.Context, tag string) []fl
 	return input
 }
 
+func (r LoggingReceiverSystemd) Pipelines(ctx context.Context) ([]otel.ReceiverPipeline, error) {
+	receiver_config := map[string]any{
+		"start_at": "beginning",
+		"priority": "debug",
+	}
+
+	modify_fields_processors, err := LoggingProcessorModifyFields{
+		Fields: map[string]*ModifyField{
+			`severity`: {
+				CopyFrom: "jsonPayload.PRIORITY",
+				// This mapping sets SeverityText in otel logs data model which naming differs from Cloud Logging model.
```

Comment:
We should be accepting the Cloud Logging values in severityText. Users can be writing this field directly, either with their own modify_fields or directly with JSON, and they can filter on those values. I think we should try to get this fixed in the exporter.

Note that the OTel spec allows (and encourages) arbitrary strings as severityText: https://opentelemetry.io/docs/specs/otel/logs/data-model/#severity-fields

For now I'd put this back to LogSeverity, change the test to only use severities where the name matches, and then file a bug to add support for the rest of the severities in the exporter.

Reply:
Done. I created this b/400435104 and i'll start working on updating the `googlecloudexporter`.